tests/validation.js

```diff
@@ -1,37 +1,19 @@
 import test from 'ava';
 import validation from '../lib/validation';
 
+
 const input = {
   target: {
     name: 'url',
-    value: 'foo bar baz',
-  },
-  all: {
-    url: 'foo bar baz',
+    value: 'ibm.com',
   },
 };
 
-const settings = {
-  target: {
-    empty: false,
-  },
-  all: {
-    url: {
-      empty: false,
-    },
-  },
-};
-
-
 // Valid input
 test('valid input', t => {
-  t.true(validation(input, settings), 'Valid input returns true');
+  t.true(validation(input), 'Valid input returns true');
 });
 
-// Invalid input
-test('validate correct input', t => {
-  const ip = input;
-  ip.target.value = '';
-
-  t.is(validation(ip, settings), 'url cannot be left blank!', 'Return string if not valid');
+test('validate text is url', t => {
+  t.true(validation(input), 'Must be an actual url');
```

Comment:
@rachelnicole please write a failing test. When you look at your code coverage:
https://travis-ci.org/punchcard-cms/input-plugin-url/builds/132883073
You can see in the nyc-coverage chart that line number 26 is not tested:
```
-----------------------|----------|----------|----------|----------|----------------|
File                   |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
-----------------------|----------|----------|----------|----------|----------------|
 input-plugin-url/     |      100 |      100 |      100 |      100 |                |
  index.js             |      100 |      100 |      100 |      100 |                |
 input-plugin-url/lib/ |       80 |       50 |      100 |       80 |                |
  validation.js        |       80 |       50 |      100 |       80 |             26 |
-----------------------|----------|----------|----------|----------|----------------|
All files              |    85.71 |       50 |      100 |    85.71 |                |
-----------------------|----------|----------|----------|----------|----------------|
```
line 26 is this line:
```
    return 'Not a valid url.';
```

so we need a test to check strings which are not urls, which will then ensure that the validation works as expected for both valid and invalid urls.